pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -75,6 +82,41 @@ func NewPlugin() plugin.Plugin {
 	return mixerplugin{}
 }
 
+func (mixerplugin) OnPreComputePerRouteFilterConfig(env *model.Environment, proxy *model.Proxy, push *model.PushContext) map[string]*types.Struct {
+	mixerPerRouteFilterConfig := make(map[string]*types.Struct)
+	for _, service := range push.Services(proxy) {
+		// Use hostname for key
+		hostname := string(service.Hostname)
+		//calculate outbound
+		attrs := addDestinationServiceAttributes(make(attributes), push, service.Hostname)
+		config := &mccpb.ServiceConfig{
+			DisableCheckCalls: true,
+			MixerAttributes:   &mpb.Attributes{Attributes: attrs},
+			ForwardAttributes: &mpb.Attributes{Attributes: attrs},
+		}
+		filterConfig := util.MessageToStruct(config)
+		key := strings.Join([]string{hostname, "outbound", policyCheckDisable}, "|")
+		mixerPerRouteFilterConfig[key] = filterConfig
+		config = &mccpb.ServiceConfig{
+			DisableCheckCalls: false,
+			MixerAttributes:   &mpb.Attributes{Attributes: attrs},
+			ForwardAttributes: &mpb.Attributes{Attributes: attrs},
+		}
+		filterConfig = util.MessageToStruct(config)
+		key = strings.Join([]string{hostname, "outbound", policyCheckEnable}, "|")
+		mixerPerRouteFilterConfig[key] = filterConfig
+		//calculate inbout
```

Comment:
This is suspicious. We should be able to disable inbound filter config, too, with an annotation. Why is it not pre-computed?

Reply:
add enable/disable for inbound in cache as well